packages/trpc/server/routers/viewer/admin/setSMSLockState.handler.ts

```diff
@@ -16,17 +16,20 @@ type GetOptions = {
 const setSMSLockState = async ({ input }: GetOptions) => {
   const { userId, username, teamId, teamSlug, lock } = input;
   if (userId) {
+    const userToUpdate = await prisma.user.findUnique({ where: { id: userId } });
+    if (!userToUpdate) throw new TRPCError({ code: "BAD_REQUEST", message: "User not found" });
+    const userUpdateInput = {
+      smsLockState: lock ? SMSLockState.LOCKED : SMSLockState.UNLOCKED,
+      ...(userToUpdate.smsLockState === SMSLockState.REVIEW_NEEDED && !lock
+        ? { smsLockReviewedByAdmin: true }
+        : {}),
+    };
     const updatedUser = await prisma.user.update({
       where: {
         id: userId,
       },
-      data: {
-        smsLockState: lock ? SMSLockState.LOCKED : SMSLockState.UNLOCKED,
-      },
+      data: userUpdateInput,
```

Comment:
```suggestion
      data: {
        smsLockState: lock ? SMSLockState.LOCKED : SMSLockState.UNLOCKED,
        smsLockReviewedByAdmin: true,
      },
```

setSMSLockState is a privileged endpoint only accessible by admin, so you just have to update the reviewedByAdmin to true.

Reply:
Done.